src/claviska/SimpleImage.php

```diff
@@ -565,6 +565,45 @@ public function crop($x1, $y1, $x2, $y2) {
     return $this;
   }
 
+  //
+  // Crops the image to fit inside a specific width and height as best as possible.
+  //
+  // $dstw - Output crop width.
+  // $dsth - Output crop height.
+  //
+  // Returns a SimpleImage object.
+  //
+  public function bestCrop($dstw, $dsth){
+    $w = $this->getWidth();
+    $h = $this->getHeight();
+    // Calculate the new width and height based on the aspect ratio of the provided width and height to fit as best
+    // possible inside the original image.
+    if($w > $h) {
+      $new_height = $dsth;
```

Comment:
Can we use `$newHeight`, `$newWidth`, etc. instead of `$new_height`, `$new_width`, etc.?